internal/pkg/cache/redis/redis.go

```diff
@@ -29,6 +29,11 @@ type Redis struct {
 	c redis.UniversalClient
 }
 
+const (
+	redisClusterPrefix = "cluster/"
```

Comment:
I do not like the implicit prefix hidden in the connection string.

I prefer if you provide two explicit New functions, one for creating a redis server and another for creating a redis cluster.